firestore/angular-rewrite/src/app/submit-review-modal/submit-review-modal.component.html

```diff
@@ -0,0 +1,49 @@
+<!--
+   * Copyright 2023 Google LLC
+   *
+   * Licensed under the Apache License, Version 2.0 (the "License");
+   * you may not use this file except in compliance with the License.
+   * You may obtain a copy of the License at
+   *
+   *      http://www.apache.org/licenses/LICENSE-2.0
+   *
+   * Unless required by applicable law or agreed to in writing, software
+   * distributed under the License is distributed on an "AS IS" BASIS,
+   * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   * See the License for the specific language governing permissions and
+   * limitations under the License.
+-->
+
+<div class="add-dialog-container">
+    <h2>Add a review!</h2>
+
+    <div id="star-rating-container">
+        <i hidden class="template material-icons star-rating" 
+        [style.color]="review.rating >= 1 ? '#feb22c' : 'gray'"
```

Comment:
Can we avoid this repetition by using a function instead?

Reply:
Function added in an upcoming commit!